pkg/sentry/arch/arch_x86.go

```diff
@@ -203,6 +203,12 @@ func (s *State) StateData() *State {
 	return s
 }
 
+// TLSPointer returns the TLS address.
+// Noop on x86.
```

Comment:
Put a similar note here about fs_base?

Reply:
If we use two step method, this is not needed!